### DISCUSSION TITLE
Can the automator accept input values from a CLI bash script?

### DISCUSSION THREAD

**Discussion by dachtera** [SAP-archive/btp-setup-automator#286] [Q&A] [answered]
Hello - in our process, we need to create multiple sub-accounts in one process.  Instead of manually changing the parameters file each time for the sub-account name, etc, can we gather these values input from the user within a simple bash script (which kicks off the automator script) to contain things like the customer directory, sub-account name, region, landscape etc to help eliminate manual editing of the .json files?

**Comment by lechnerc77** [comment, marked as answer]
Yes it can. This section in the documentation gives you an idea to see the parameters and transfer them: <https://github.com/SAP-samples/btp-setup-automator/blob/main/docs/README.md#available-parameters>.

Example:
```bash
btpsa -myemail 'test@test.com' -globalaccount 'ID of the account' 
``` 
Having said that I think that using scripts has downsides compared to a version-controlled declaration of the setup via usecase files.
